pkg/ingester-rf1/metastore/metastore_state_add_block.go

```diff
@@ -11,8 +11,8 @@ import (
 )
 
 func (m *Metastore) AddBlock(_ context.Context, req *metastorepb.AddBlockRequest) (*metastorepb.AddBlockResponse, error) {
-	_, resp, err := applyCommand[*metastorepb.AddBlockRequest, *metastorepb.AddBlockResponse](m.raft, req, m.config.Raft.ApplyTimeout)
-	return resp, err
+	_, resp, err := applyCommand(m.raft, req, m.config.Raft.ApplyTimeout)
```

Comment:
why were the generics removed from this method? I don't mind, just curious

Reply:
This should be `m.applyCommand` 😄 I had to remove the generics because otherwise it doesn't compile when storing the function pointer in the struct (used for tests).